docs/contributing/building/nocore-ebpf.md

```diff
@@ -118,36 +118,38 @@ If you install the non CO-RE eBPF object and run **tracee-ebpf** in an
 environment that needs it, then the debug output will look like:
 
 ```text
-$ sudo ./dist/tracee-ebpf --debug
-
-OSInfo: ARCH: x86_64
-OSInfo: VERSION: "20.04.3 LTS (Focal Fossa)"
-OSInfo: ID: ubuntu
-OSInfo: ID_LIKE: debian
-OSInfo: PRETTY_NAME: "Ubuntu 20.04.3 LTS"
-OSInfo: VERSION_ID: "20.04"
-OSInfo: VERSION_CODENAME: focal
-OSInfo: KERNEL_RELEASE: 5.8.0-63-generic
-BTF: bpfenv = false, btfenv = false, vmlinux = false
-BPF: no BTF file was found or provided, trying non CO-RE eBPF at
-     /tmp/tracee/tracee.bpf.5_8_0-63-generic.v0_6_5-20-g3353501.o
+$ sudo ./dist/tracee-ebpf --log debug
+{"level":"debug","ts":1670617146.8835983,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"VERSION_ID","OS_KERNEL_RELEASE":"\"20.04\""}
+{"level":"debug","ts":1670617146.8852425,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"VERSION_CODENAME","OS_KERNEL_RELEASE":"focal"}
+{"level":"debug","ts":1670617146.885269,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"KERNEL_RELEASE","OS_KERNEL_RELEASE":"5.4.0-132-generic"}
+{"level":"debug","ts":1670617146.8852894,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"ARCH","OS_KERNEL_RELEASE":"x86_64"}
+{"level":"debug","ts":1670617146.8853118,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"VERSION","OS_KERNEL_RELEASE":"\"20.04.5 LTS (Focal Fossa)\""}
+{"level":"debug","ts":1670617146.8853261,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"ID","OS_KERNEL_RELEASE":"ubuntu"}
+{"level":"debug","ts":1670617146.8853502,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"ID_LIKE","OS_KERNEL_RELEASE":"debian"}
+{"level":"debug","ts":1670617146.885369,"msg":"osinfo","pkg":"urfave","file":"urfave.go","line":37,"OSReleaseField":"PRETTY_NAME","OS_KERNEL_RELEASE":"\"Ubuntu 20.04.5 LTS\""}
```

Comment:
Can you please remind me why do we have the extra OS_KERNEL_RELEASE in all of the above messages?

Reply:
This is strange. I'll check it.

Reply:
This is a snapshot from a different environment; fixing. Thanks.

Reply:
`OS_KERNEL_RELEASE` was being used as a log field. I'm making it clearer.

```log
{"level":"debug","ts":1670972052.3996286,"msg":"osinfo","VERSION_CODENAME":"focal","KERNEL_RELEASE":"5.4.0-91-generic","ARCH":"x86_64","VERSION":"\"20.04.5 LTS (Focal Fossa)\"","ID":"ubuntu","ID_LIKE":"debian","PRETTY_NAME":"\"Ubuntu 20.04.5 LTS\"","VERSION_ID":"\"20.04\"","pkg":"urfave","file":"urfave.go","line":53}
```

Reply:
Done.